tests/platform_tests/api/test_thermal.py

```diff
@@ -111,45 +135,49 @@ def test_get_temperature(self, duthost, localhost, platform_api_conn):
     def test_get_low_threshold(self, duthost, localhost, platform_api_conn):
         # Ensure the thermal low threshold temperature is sane
         for i in range(self.num_thermals):
-            temperature = thermal.get_low_threshold(platform_api_conn, i)
+            low_threshold = thermal.get_low_threshold(platform_api_conn, i)
 
-            if self.expect(temperature is not None, "Unable to retrieve Thermal {} low threshold".format(i)):
-                if self.expect(isinstance(temperature, float), "Thermal {} low threshold appears incorrect".format(i)):
-                    self.expect(temperature > 0 and temperature <= 100,
-                                "Thermal {} low threshold {} reading is not within range".format(i, temperature))
+            if self.expect(low_threshold is not None, "Unable to retrieve Thermal {} low threshold".format(i)):
+                if self.expect(isinstance(low_threshold, float), "Thermal {} low threshold appears incorrect".format(i)):
+                    self.expect(low_threshold > 0 and low_threshold <= 100,
+                                "Thermal {} low threshold {} reading is not within range".format(i, low_threshold))
+                    self.compare_value_with_platform_facts('low_threshold', low_threshold, i)
         self.assert_expectations()
 
     def test_get_high_threshold(self, duthost, localhost, platform_api_conn):
         # Ensure the thermal high threshold temperature is sane
         for i in range(self.num_thermals):
-            temperature = thermal.get_high_threshold(platform_api_conn, i)
+            high_threshold = thermal.get_high_threshold(platform_api_conn, i)
 
-            if self.expect(temperature is not None, "Unable to retrieve Thermal {} high threshold".format(i)):
-                if self.expect(isinstance(temperature, float), "Thermal {} high threshold appears incorrect".format(i)):
-                    self.expect(temperature > 0 and temperature <= 100,
-                                "Thermal {} high threshold {} reading is not within range".format(i, temperature))
+            if self.expect(high_threshold is not None, "Unable to retrieve Thermal {} high threshold".format(i)):
+                if self.expect(isinstance(high_threshold, float), "Thermal {} high threshold appears incorrect".format(i)):
+                    self.expect(high_threshold > 0 and high_threshold <= 100,
+                                "Thermal {} high threshold {} reading is not within range".format(i, high_threshold))
+                    self.compare_value_with_platform_facts('high_threshold', high_threshold, i)
```

Comment:
For now we are not storing threshold values in platform.json. Please remove this check.

Reply:
fixed